Dockerfile

```diff
@@ -5,6 +5,7 @@ WORKDIR /app
 COPY protocol/ protocol/
 COPY sdk/ sdk/
 COPY lib/ocrypto lib/ocrypto
+COPY lib/fixtures lib/fixtures
```

Comment:
Is our build dependent on fixtures package?

Reply:
yes, unit tests need it in order to generate test data

Reply:
I see what you did now 😄